ivy/functional/frontends/torch/nn/functional/pooling_functions.py

```diff
@@ -106,6 +106,44 @@ def avg_pool2d(
         data_format=data_format,
     )
 
+@to_ivy_arrays_and_back
+def avg_pool3d(
```

Comment:
`ceil_mode`, `count_include_pad`, `divisor_override` these are not implemented in the backend yet, the frontend function needs the complete backend implementation(superset), If the backend task is open I'd suggest work on the superset behavior or else wait for it to be implemented by someone. :slightly_smiling_face: 
In the meanwhile you are free to work on any other frontend function. :slightly_smiling_face: